docs/beginner_guide.rst

```diff
@@ -0,0 +1,46 @@
+:orphan:
+
+.. _beginner_guide:
+
+
+Beginner Guide
+==============
+
+Welcome to the beginners guide for JAX. 
+On this page we will introduce you to the key ideas of Jax,
+show you how to get Jax running
+and provide you some tutorials to get started.
+
+If looking to jump straight in take a look at the jax quickstart.
+
+.. toctree::
```

Comment:
I couldnt figure out how to create a direct reference to a .ipynb file from sphinx so settling for this toc tree instead for now

Reply:
Yeah, we would need myst-md anchors to reference notebook sections (e.g. `(reference-name)=`) but unfortunately these are not hidden in the notebook view. So elsewhere we tend to just refer to URLs, which mostly never go stale.

Using a TOC is fine here.

Reply:
Thanks